app/services/decisions/client_decision_tree.rb

```diff
@@ -40,7 +40,7 @@ def after_contact_details
           form_object.applicant
         )
       else
-        show('/home', action: :index)
+        show('/steps/case/urn', action: :edit)
```

Comment:
As this is an `edit` step it is best to do the helper edit method `edit('/steps/case/urn')` although in this case is functionally equivalent.

Reply:
Oh of course, I wasn't thinking. Thank you